Composer/plugins/vacore/template/schemas/update-schema.ps1

```diff
@@ -0,0 +1,30 @@
+param (
```

Comment:
Do we need to ship these scripts with each template? They are already in the runtime with composer?